src/org/labkey/test/pages/ReactAssayDesignerPage.java

```diff
@@ -359,6 +379,13 @@ final Optional<OptionSelect<MetadataInputFormat>> metadataInputSelect()
             return OptionSelect.finder(Locator.id("assay-design-selectedMetadataInputFormat"), MetadataInputFormat.class)
                     .findOptional(propertiesPanel);
         }
+        Locator runOnActionCheckboxLoc(String fileName, String actionLabel) // actionLabel is Run on Import, or Run on Edit
```

Comment:
Make an `enum` for the possible actions. Tests shouldn't need to know the exact text for the checkbox labels.